### PR TITLE
DOP-3281 - Fix tooltip font sizing

### PR DESCRIPTION
**Pull request by carooobregon**
**Ticket**
[DOP-3281](https://jira.mongodb.org/browse/DOP-3281)

**Description**
LeafyGreen's tooltip component doesn't have a modification to specify the font-size of the tooltip's text but rather uses the `baseFontSize` to use as font size. Out of the components we're currently using, `Tooltip` is the only one that doesn't have a customizable font-size so changed the `baseFontSize` and then adjusted the other instances of the site that were using `baseFontSize` set as 16px.

### CODE REVIEW

**Review by mmeigs — COMMENTED** [review]
I dropped a few nits in! And also... there's a console error being thrown :( 

"Invalid prop `baseFontSize` of value `13` supplied to `LeafyGreenProvider`, expected one of [14,16]."

So, maybe we ask Allison if 14 is ok for the tooltip? Although I wonder if that screws up anything else that might take its font from baseFontSize...? Maybe worth reaching out to leafygreen-ui to ask if there's a way to overwrite the font-size in the tooltip. Maybe they have an idea. (first search their channel to see if anyone's asked before!! Sometimes happens)

File: frontend/src/components/Homepage.tsx [hunk 1]
```diff
@@ -6,6 +6,10 @@ const headerStyles = css`
   margin: 0 0 24px 0;
 `;
 
+const homepageTextStyle = css`
```
[the remark this answers is not in the document]
Comment (carooobregon) [reply frontend/src/components/Homepage.tsx:9]: good idea!

File: frontend/src/components/Repo.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { DropResult } from 'react-beautiful-dnd';
 import { H3 } from '@leafygreen-ui/typography';
 import { Body } from '@leafygreen-ui/typography';
 import { palette } from '@leafygreen-ui/palette';
-import { css, cx } from '@leafygreen-ui/emotion';
+import { css as LeafyCSS, cx } from '@leafygreen-ui/emotion';
```
[the remark this answers is not in the document]
Comment (carooobregon) [reply frontend/src/components/Repo.tsx:7]: thanks for the catch!

**Comment by mmeigs** [comment]
> I dropped a few nits in! And also... there's a console error being thrown :(
> 
> "Invalid prop `baseFontSize` of value `13` supplied to `LeafyGreenProvider`, expected one of [14,16]."
> 
> So, maybe we ask Allison if 14 is ok for the tooltip? Although I wonder if that screws up anything else that might take its font from baseFontSize...? Maybe worth reaching out to leafygreen-ui to ask if there's a way to overwrite the font-size in the tooltip. Maybe they have an idea. (first search their channel to see if anyone's asked before!! Sometimes happens)

@carooobregon I don't know if you saw this ^^

**Comment by carooobregon** [comment]
> > I dropped a few nits in! And also... there's a console error being thrown :(
> > "Invalid prop `baseFontSize` of value `13` supplied to `LeafyGreenProvider`, expected one of [14,16]."
> > So, maybe we ask Allison if 14 is ok for the tooltip? Although I wonder if that screws up anything else that might take its font from baseFontSize...? Maybe worth reaching out to leafygreen-ui to ask if there's a way to overwrite the font-size in the tooltip. Maybe they have an idea. (first search their channel to see if anyone's asked before!! Sometimes happens)
> 
> @carooobregon I don't know if you saw this ^^

Sorry i hadnt seen this! Just asked Allison what she thinks about the 14 px size and will update accordingly

**Comment by carooobregon** [comment]
@mmeigs allison reviewed this and said its fine keeping it 14px! :)

**Review by mmeigs — APPROVED** [review]
LGTM!

**Pull request merged**